scripts/install_vim.sh

```diff
@@ -1,88 +1,186 @@
 #!/bin/bash
 
 set -e
+set -x
```

Comment:
Is this added so the builds are verbose or is this left in by mistake?  Maybe this should be enabled if a `VERBOSE` env variable exists.